nss-day-cohort-50/nashville-kennels-team-closeau
# Mc animal form

# Description
1. In `AnimalForm.js`, imported `EmployeeRepository`
2. State was already defined for employees, but there was no modification to state, used `useEffect()` hook to make a HTTP request (using function from `EmployeeRepository` and `setEmployees`. This made iteration of the employees array possible in the form. 
3. Last step to `constructNewAnimal()` was `.then(() => props.history.push(“/animals”), which gave an error. I took off the `props.` part, imported `useHistory()` hook from `react-router-dom`, and stored `useHistory()` in `history` variable. This resolved the issue.
4. LocationId was not getting saved to API, added property `employeeLocations[0]` before `.locationId`.

## Steps to test
1. In browser, with devtools open, go to Animals, then click `Register Animal`. 
2. Populate all the fields
3. Make sure the HTTP request “POST” has all the information needed.
